internal/config/config.go

```diff
@@ -107,9 +115,5 @@ func GetRegistry(cfg Config, registryName string) (Registry, error) {
 		return Registry{}, errors.Errorf("registry %s is not defined in your config file", style.Symbol(registryName))
```

Comment:
Wouldn't this fail if I call `GetRegistry(cfg, "official")`? Where does the amendment of the in-memory "official" happen?

Reply:
This code path that you're referring to is for the `register-buildpack` command.  If there isn't a defined registry in `config.toml` file, `GetRegistry()` will just return a single default registry to use for opening up the Github issue against.    As of now, if I called `GetRegistry(cfg, "official")` it would look for "official", but it shouldn't find it, and will return the hardcoded default registry which would be the `official` registry.

There isn't any amending that happening.  This is only for when no registries are defined, then it will default to "official".

Reply:
We aren't doing any amending, never had before either.  This code path is responsible for returning a single registry that should be used for creating the Github issue against.  When a user runs `register-buildpack` if no registries are defined, a default registry will be used instead, which is the `"official"` registry.